sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroidOptions.java

```diff
@@ -32,6 +34,25 @@
   /** Enable or disable automatic breadcrumbs for App Components Using ComponentCallbacks */
   private boolean enableAppComponentBreadcrumbs = true;
 
+  public SentryAndroidOptions() {
+    setSdkVersion(createSdkVersion());
+  }
+
+  private @NotNull SdkVersion createSdkVersion() {
+    final SdkVersion sdkVersion = new SdkVersion();
+
+    sdkVersion.setName("sentry.java.android");
+    String version = BuildConfig.VERSION_NAME;
+    sdkVersion.setVersion(version);
+
+    // add 2 default packages
+    sdkVersion.addPackage("maven:sentry-android-core", version);
+    // TODO: sentry-core should add itself as the version may mismatch
+    sdkVersion.addPackage("maven:sentry-core", version);
```

Comment:
Created issue to follow up: https://github.com/getsentry/sentry-java/projects/1#card-42202340